src/SimpleServiceManager.sol

```diff
@@ -58,11 +58,11 @@ contract SimpleServiceManager is ISimpleServiceManager, Initializable, OwnableUp
     /// @dev Set of currently registered operator addresses
     EnumerableSet.AddressSet private registeredOperators;
 
-    /// @notice Maps a signing key to its associated operator address
-    mapping(address => address) public signingKeyToOperatorAddress;
+    /// @notice Maps a signing key to its associated registration key (operator address)
+    mapping(address => address) public signingKeyToRegistrationKey;
```

Comment:
Completely understand the urge to change the variable name. 

However, if we're going to change the name here we should probably change the name in the ServiceManager for consistency.